### PR TITLE
update cpms check steps to prevent blocking tests

### PR DESCRIPTION
**Pull request by WenXinWei**
there is no workspace under cpms now, so the check failed. to prevent bolcking test for other teams. let's temporarily remove cpms check.
and i will confirm with dev later. cc @sgaoshang @jinyunma

### CODE REVIEW

**Comment by WenXinWei** [comment]
please help review when you have time @jianlinliu @jinyunma

**Comment by jinyunma** [comment]
> there is no workspace under cpms now,

Searched in installer repo, it should be expected now, see related PR https://github.com/openshift/installer/pull/8354, and [verification](https://issues.redhat.com//browse/OCPBUGS-32947) in OCPBUGS-32947.

I guess need to update step to remove check for this part.

**Comment by WenXinWei** [comment]
> > there is no workspace under cpms now,
> 
> Searched in installer repo, it should be expected now, see related PR [openshift/installer#8354](https://github.com/openshift/installer/pull/8354), and [verification](https://issues.redhat.com/browse/OCPBUGS-32947?focusedId=24757338&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24757338) in OCPBUGS-32947.
> 
> I guess need to update step to remove check for this part.

got it. thanks.

**Comment by WenXinWei** [comment]
I removed singlezone check function. is that looks good to you @jinyunma ?

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-tp-f28

File: ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh [hunk 1]
```diff
@@ -81,9 +62,7 @@ if (( ${ocp_minor_version} > 15 )) || [[ "${FEATURE_SET}" == "TechPreviewNoUpgra
       checkTemplate
       INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
       readarray -t zones_setting_from_config < <(yq-go r ${INSTALL_CONFIG} 'controlPlane.platform.vsphere.zones[*]')
-      if (( ${#zones_setting_from_config[@]} > 1 )); then
-          checkMultizone
-      else
-	  checkSinglezone
+      if [[ "${FEATURE_SET}" != "TechPreviewNoUpgrade" ]]; then
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:65]: why we need this condition?

Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:65]: To make the if log be more clear, how about:
```
if (( ${ocp_minor_version} < 15 )); then
	echo "CPMS failureDomain check is only available on 4.15+ cluster, skip the check!"
        exit 0
fi
if (( ${ocp_minor_version} == 15 )) && [[ "${FEATURE_SET}" != "TechPreviewNoUpgrade" ]]; then
	echo "CPMS failureDomain check is only available when TechPreviewNoUpgrade enabled on 4.15 cluster, skip the check!"
        exit 0
fi
echo "cpms spec:"
oc get controlplanemachineset -n openshift-machine-api -ojson | jq -r '.items[].spec.template.machines_v1beta1_machine_openshift_io'
checkTemplate
if (( ${ocp_minor_version} > 15 )); then
    checkCpms
fi
```

File: ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
 
 
-function checkMultizone() {
+function checkcpms() {
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:11]: pls also change the following `INFO` message in this function to remove the `multi` keyword.

Comment (WenXinWei) [reply ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:11]: got it.

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-tp-f28

**Comment by wewang58** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-tp-f28-destructive

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-tp-f28-destructive

**Comment by jianlinliu** [comment]
/lgtm

leave this to @jinyunma for more comments.

**Comment by jinyunma** [comment]
seems that job still failed at step of cpms check. 
'''
ERROR: The failure domain name are different between install_config and cmps. please check
'''

File: ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh [hunk 3]
```diff
@@ -75,15 +56,13 @@ if (( ${ocp_minor_version} < 15 )); then
 	echo "CPMS failureDomain check is only available on 4.15+ cluster, skip the check!"
         exit 0
 fi
-if (( ${ocp_minor_version} > 15 )) || [[ "${FEATURE_SET}" == "TechPreviewNoUpgrade" ]]; then
-      echo "cpms spec:"
-      oc get controlplanemachineset -n openshift-machine-api -ojson | jq -r '.items[].spec.template.machines_v1beta1_machine_openshift_io'
-      checkTemplate
-      INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
-      readarray -t zones_setting_from_config < <(yq-go r ${INSTALL_CONFIG} 'controlPlane.platform.vsphere.zones[*]')
```
Comment (jinyunma) [inline ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:83]: This line seems still required for failureDomain name check.

Comment (WenXinWei) [reply ci-operator/step-registry/cucushift/installer/check/vsphere/cpms/cucushift-installer-check-vsphere-cpms-commands.sh:83]: yes, It was originally written for multiple zones.  i will update.

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-tp-f28-destructive

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-ipi-proxy-f14-disasterrecovery

**Pull request merged**